projects/utils/src/api/git/api.ts

```diff
@@ -0,0 +1,35 @@
+import { StatusEvents } from '../../types'
+
+export interface IGitSystem {
+  events: {
+    //TODO:
```

Comment:
As there is no events expected in a git plugin, you can remove TODO

Reply:
This has been deleted